db/migrate/20190617165412_rename_phpipam_id_to_external_ipam_id_in_subnets.rb

```diff
@@ -0,0 +1,9 @@
+class RenamePhpipamIdToExternalIpamIdInSubnets < ActiveRecord::Migration[5.2]
+  def up
+    rename_column :subnets, :phpipam_id, :external_ipam_id
+  end
+
+  def down
+    rename_column :subnets, :external_ipam_id, :phpipam_id
+  end
+end
```

Comment:
Please squash the two migrations into one, just add "external_ipam_id" column to the subnet.